modules/custom/az_card/src/Plugin/Field/FieldFormatter/AZCardDefaultFormatter.php

```diff
@@ -152,6 +149,30 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
             }
             $column_classes[] = $card_defaults['card_width'] ?? 'col-md-4 col-lg-4';
             $card_classes = $card_defaults['card_style'] ?? 'card';
+
+            // Is the card clickable?
+            if (isset($card_defaults['card_clickable']) && $card_defaults['card_clickable']) {
+              $link_render_array['#attributes']['class'][] = 'stretched-link';
+              $card_classes .= ' shadow mb-4';
```

Comment:
```suggestion
              $card_classes .= ' shadow';
```
Unnecessary since the column div already has padding on it